File: beginner/javascript/2143.js
var input = require("fs").readFileSync("stdin", "utf8");
var lines = input.split("\n");

var t, n;

t = parseInt(lines.shift());

while (t != 0) {
    for (var i = 1; i <= t; i++) {
        n = parseInt(lines.shift());
        if (n % 2 != 0) 
            console.log((n * 2) - 1);
        else 
            console.log((n * 2) - 2);
    }
    
    t = parseInt(lines.shift());
}

File: beginner/python/2143.py
t = int(input())

while (t != 0):
    for i in range(1,t+1):
        n = int(input())
        if (n % 2 != 0):
            print((n * 2) - 1)
        else:
            print((n * 2) - 2)
    
    t = int(input())
